packages/legend-extension-application-studio-query-builder/src/components/MappingExecutionQueryBuilder.tsx

```diff
@@ -64,6 +64,11 @@ export const MappingExecutionQueryBuilder = observer(
         queryBuilderExtension.queryBuilderState.initialize(
           executionState.queryState.query,
         );
+        await Promise.all([
```

Comment:
See my comment in `QueryBuilderChangeDetectionState`, we should only need to do:
`await queryBuilderState.changeDetectionState.start();` here